### PR TITLE
Various grammatical and language fixes for the FAQ.

### PR DESCRIPTION
**Pull request by hlapp**
Some of the changes suggested here are minor (typographical and grammatical), while a few are also restatements of what I think was the intention, with the intent to improve clarity, readability, and professional presentation.

The latter category changes obviously involve my interpretation of what was there. Even if you reject those changes, I suggest you still find alternative ways to revise the respective text pieces because right now they do leave more to interpretation than they should.

### CODE REVIEW

File: 05-FAQ.md [hunk 1]
```diff
@@ -18,24 +18,19 @@ Yes ! You can:
  * [Submit](../write) paper for the work you've already replicated
  * Spread the word about Re**Science** in your community
    ([twitter](http://twitter.com/ReScienceEds), mailing lists, blogs, etc.)
- * [Star the project](https://github.com/ReScience/ReScience) on github
```
Comment (rougier) [inline 05-FAQ.md]: Maybe I added this maybe after your edit (or yo don't like the idea ?)

File: 05-FAQ.md [hunk 2]
```diff
@@ -18,24 +18,19 @@ Yes ! You can:
...
    propose modifications (through PR)
  * [Opening an issue](https://github.com/ReScience/ReScience/issues) if you
    have any unanswered question.
- * [Website](https://github.com/ReScience/rescience.github.io) would certainly
-   needs the help of a [Jekyll](http://jekyllrb.com) expert and the eye of a
+ * The [Website](https://github.com/ReScience/rescience.github.io) could certainly
+   use the help of a [Jekyll](http://jekyllrb.com) expert and the eye of a
    designer (but keep in mind we want to keep a simple & clean website where
    content is the primary focus)...
- * Print stickers from the
```
Comment (rougier) [inline 05-FAQ.md]: Same

**Comment by rougier** [comment]
Thanks a lot. Only two comment at the very beginning. Then I merge.

**Comment by rougier** [comment]
Well I can put them back later, your corrections are more important.

File: 05-FAQ.md (hunk 1 above)
Comment (hlapp) [inline 05-FAQ.md]: I didn't mean to delete this (nor the following deletion). Perhaps a mistake from editing on the Github UI. I'll see whether I can get those back in.

**Comment by hlapp** [comment]
I see - you merged more quickly than I could act :smile: I assume you'll put those two lines back in?

**Comment by rougier** [comment]
Maybe you can do it, this will save you correcting my English errors..
The sticker line is a pretext to have a link on the material.

**Pull request merged**